src/librustc_save_analysis/dump_visitor.rs

```diff
@@ -334,10 +330,14 @@ impl<'l, 'tcx: 'l, 'll, D: Dump + 'll> DumpVisitor<'l, 'tcx, 'll, D> {
                     scope: scope
                 }.lower(self.tcx));
             }
+            Def::Local(..) |
+            Def::Upvar(..) |
```

Comment:
The function is used only on exports, so everything non-exportable (locals, associated items, etc) is in the span_bug! list.